src/it/scala/org/ergoplatform/it/DeepRollBackSpec.scala

```diff
@@ -81,7 +81,7 @@ class DeepRollBackSpec extends FreeSpec with IntegrationSuite {
       minerBBestBlock shouldEqual minerABestBlock
     }
 
-    Await.result(result, 15.minutes)
+    Await.result(result, 25.minutes)
```

Comment:
Looks like something wrong with it-tests, which will make them less robust over time.

Reply:
checking that still, to be resolved in 3.3.0 branch maybe